default/python/AI/MilitaryAI.py

```diff
@@ -845,10 +866,26 @@ def assign_military_fleets_to_systems(use_fleet_id_list=None, allocations=None,
 
 @cache_by_turn
 def get_tot_mil_rating():
+    """
+    Give an assessment of total miltary rating considering all fleets as if distributed to separate systems.
```

Comment:
Add blank line.

Reply:
The blank line is added, Github just doesn't figure out that the comment is outdated because of which lines were selected for the comment versus which lines changed.